deploy/cr.yaml

```diff
@@ -22,6 +22,10 @@ spec:
 #    livenessDelaySec: 300
 #    forceUnsafeBootstrap: false
 #    configuration: |
+#      [sst]
+#      xbstream-opts=--decompress
+#      [xtrabackup]
+#      compress=lz4
```

Comment:
could you please move it after `[mysqld]` section

Reply:
Moved.